docs/API-access-tokens.md

```diff
@@ -0,0 +1,23 @@
+# API access tokens
+
+To use Inductiva API you will need a valid API token. At the moment, you can request an API token [here](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform). 
+
+Once you have your API token, you can either:
+- set the `INDUCTIVA_API_KEY` as an environment variable in your terminal as follows
+
+```bash
+export INDUCTIVA_API_KEY="YOUR_API_KEY"
+```
+
+- set the `api_key` attribute of the `inductiva` module in your Python script as follows
```

Comment:
I would like to deprecate this method on this release. It bad practice to have keys in the code.

@sergiomsantos @IvanPombo can we remove all examples where we do this?

As a plus, our exemples can be run automatically because there will be no need to substitute the key in the code snippet.